app/data_models/app_models.py

```diff
@@ -58,7 +58,7 @@ class QuestionnaireStateSchema(Schema, DateTimeSchemaMixin):
     version = fields.Integer()
 
     @post_load
-    def make_model(self, data):
+    def make_model(self, data, many, **kwargs):
```

Comment:
Based on the examples in the doc, it doesn't look like `many` is needed when `pass_many=False` which is the default as it is followed by `**kwargs`.

Reply:
It shouldn't be, but I got an error:
```
strip_whitespace() got an unexpected keyword argument 'many'
```